libs/vkd3d/state.c

```diff
@@ -580,9 +516,15 @@ static HRESULT d3d12_root_signature_init_root_descriptor_tables(struct d3d12_roo
 
         table = &root_signature->parameters[i].u.descriptor_table;
         range_count = p->u.DescriptorTable.NumDescriptorRanges;
+        range_descriptor_offset = 0;
 
         root_signature->parameters[i].parameter_type = p->ParameterType;
         table->range_count = range_count;
+        table->table_index = t++;
```

Comment:
Is this used so that when you call SetDescriptorTable() you can transform parameter_index to push constant offset?

Reply:
Yes.